lively.collab/comments/commentIndicator.js

```diff
@@ -64,6 +63,15 @@ export class CommentIndicator extends Label {
     }
   }
 
+  display () {
```

Comment:
maybe we make a `visibility` section for this and hide? We could put `abandon` into it as well or move `abandon` into a category `creation/deletion` that has `connectMorphs` and `canBeCopied` in it